pkgs/development/python-modules/tensorboard-plugin-wit/default.nix

```diff
@@ -0,0 +1,31 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  version = "1.7.0";
+  pname = "tensorboard_plugin_wit";
+  format = "wheel";
+
+  disabled = ! isPy3k;
+  
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    abi = "none";
+    platform = "any";
+    sha256 = "0nv855qm2fav70lndsrv810pqgg41sbmd70fk86wk18ih825yxzf";
+  };
+
+  # Tests require too many extra dependencies
+  doCheck = false;
```

Comment:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.